pkg/agent/nodeportlocal/k8s/controller.go

```diff
@@ -69,19 +76,22 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 	for i := 0; i < numWorkers; i++ {
 		go wait.Until(c.Worker, time.Second, stopCh)
 	}
-	c.Ctrl.Run(stopCh)
-	cache.WaitForCacheSync(stopCh, c.Ctrl.HasSynced)
+
+	go c.PodController.Run(stopCh)
+	go c.SvcController.Run(stopCh)
+	cache.WaitForCacheSync(stopCh, c.PodController.HasSynced)
+	cache.WaitForCacheSync(stopCh, c.SvcController.HasSynced)
```

Comment:
please use `WaitForNamedCacheSync`

Reply:
Done, I have specified the controllerName to be `"AntreaAgentNPLController"`